components/form/index.js

```diff
@@ -22,6 +23,7 @@ const SuccessHeader = styled.h2`
 
 const Form = () => {
   const [isFormSubmitted, setIsFormSubmitted] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
```

Comment:
Not really loading the form, user is submitting the form. How about `isSubmitting` and `setIsSubmitting`?

Reply:
yes I think isSubmitting, setIsSubmitting is better and more consistent